### ISSUE TITLE
Fix Composer 2 compatibility

### ISSUE THREAD

**Issue by brandonkelly** [TopShelfCraft/Scraper#2]
When installing this plugin with Composer 1, the following deprecation notice is output:

> Deprecation Notice: Class `topshelfcraft\scraper\lib\SimpleHtmlDom\simple_html_dom_node` located in `./vendor/topshelfcraft/scraper/src/lib/simple_html_dom.php` does not comply with psr-4 autoloading standard\. It will not autoload anymore in Composer v2\.0\.

Composer 2 was released on October 24, and is now the default version that will be installed, so it’s critical that this gets resolved ASAP, or people will start getting `Class not found` errors.

To fix, run the following command:

```bash
> git mv src/lib/simple_html_dom.php src/lib/SimpleHtmlDom/simple_html_dom_node.php
```

Then commit your change and [tag a new release](https://craftcms.com/docs/3.x/extend/plugin-store.html#plugin-releases).

_(Note that you must use the `git mv` command, as Git tends to not notice case-sensitive file renames otherwise.)_

**Comment by michaelrog** [comment]
Alas, a tidge more complicated; SimpleHtmlDom harkens back to the "let's cram two classes and several global functions into a single file" era of PHP libs... #goodoldays

Anyhoo, PSR-4 issues are fixed in d812abe and released as `3.0.0`!

**Comment by brandonkelly** [comment]
It was an automated issue body, yeah I can see how this one would be a bit more involved. Thanks for getting it sorted!
